script/utils/diamondEMERGENCYPauseGitHub.sh

```
@@ -116,6 +130,37 @@ function handleNetwork() {
   fi
 }
 
+function printStatus() {
+  local NETWORK="$1"
+
+  # get RPC URL for given network
+  local RPC_KEY="ETH_NODE_URI_$(tr '[:lower:]' '[:upper:]' <<<"$NETWORK")"
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)